assets/src/scss/pages/post/_comments-form.scss

```diff
@@ -1,6 +1,6 @@
 .comments-block {
   .form-section .comment-respond {
-    background: $grey-05;
+    background: var(--color-background-comments_block, $grey-05);
```

Comment:
Here can't you use `_--` or something like `--comments-block--` to automatically create a variable? If not, at least we should respect the same logic for the name, so start with the element name and then the CSS property 🙂

Reply:
Don't you think if it's better to start using tokens instead? `--color-background-comments_block` it's p4 token.

Also, I think the CSS variable implementation is going to be removed in the future whenever we have tokens already implemented. 
What do you think?